docs/integration/chip/stm32.rst

```diff
@@ -43,7 +43,7 @@ the *main.c* file. \* Create some frame buffer(s) as global variables:
   peripherals, and LCD panel, call :cpp:func:`lv_init` to initialise LVGL.
   You can then create the display driver using
   :cpp:func:`lv_disp_create`, and register the frame buffers using
-  :cpp:func:`lv_disp_set_draw_buffers`.
+  :cpp:func:`lv_disp_set_buffers`.
```

Comment:
It is recommended to replace all `lv_disp` in this PR with `lv_display`.Because I found that there is no `v_disp_set_buffers` in`lv_api_map.h`